runtime/lua/vim/lsp/rpc.lua

```diff
@@ -32,7 +32,7 @@ end
 --- Parses an LSP Message's header
 ---
 ---@param header string: The header to parse.
----@return table parsed headers
+---@return table headers parsed headers
```

Comment:
Just put `#` for this stuff. It looks like nonsense to repeat random names in the return values.
```suggestion
---@return table # parsed headers
```